crates/holochain_cascade/src/lib.rs

```diff
@@ -390,7 +454,12 @@ where
     ) -> CascadeResult<Option<EntryDetails>> {
         let authoring = self.am_i_authoring(&entry_hash.clone().into())?;
         let authority = self.am_i_an_authority(entry_hash.clone().into()).await?;
-        let query = GetEntryDetailsQuery::new(entry_hash.clone());
+        let query = match self.private_data.clone() {
+            Some(author) => {
+                GetEntryDetailsQuery::with_private_data_access(entry_hash.clone(), author)
+            }
+            None => GetEntryDetailsQuery::new(entry_hash.clone()),
+        };
```

Comment:
this change seems to be duplicate-ly applied multiple times in this file. Is it worth a function abstraction?

Reply:
Note to self this is fixed in commit `bb78fa7c3e66363ee839ca0339b5027735678d83`

Reply:
Resolved in https://github.com/holochain/holochain/commit/bb78fa7c3e66363ee839ca0339b5027735678d83